src/mame/skeleton/venteta.cpp

```diff
@@ -96,8 +103,9 @@ GFXDECODE_END
 
 void venteta_state::venteta(machine_config &config)
 {
-	ATMEGA168(config, m_maincpu, 24_MHz_XTAL); // TODO: actually ATMEGA103
+	ATMEGA1280(config, m_maincpu, 24_MHz_XTAL); // TODO: actually ATMEGA103
 	m_maincpu->set_addrmap(AS_PROGRAM, &venteta_state::program_map);
+	m_maincpu->set_addrmap(AS_PROGRAM, &venteta_state::data_map);
```

Comment:
This map should be installed for `AS_DATA` rather than overriding the previous `AS_PROGRAM` map.